### ISSUE TITLE
Import Data and Export my data buttons always disabled

### ISSUE THREAD

**Issue by Nardo86** [stephane-r/holoplay-pwa#93]
Hi and thanks for the amazing project!

I'm trying to abandon youtube music and i like to take my playlist with me,. I'm running the pwa from my docker but the website app.holoplay.io show the same problem: if i go to Settings > Export and select everything the button "Export my data" remains disabled.
I also tried to import a json configuration from invidious, the file is read correctly and i can select my playlist but the "import data" button remains disabled.

I'm on Debian 12 and tryed with Brave, Chromium and Firefox

**Comment by stephane-r** [comment]
Hello @Nardo86 :)

Do you have add your playlists on right box like this ? 

![image](https://github.com/user-attachments/assets/026b34c0-304b-4780-a71e-936c0eea4082)
![image](https://github.com/user-attachments/assets/0a9f5f1d-9c2d-41d6-985e-f9d0f6305a9e)

Tell me if it's ok. Maybe the action of transferring the playlists to the right box is not useful and less explicit.

**Comment by Nardo86** [comment]
Sorry, I didn't understood the UI, I thought the selection on the left was enough to enable the button..

So I propose a suggestion to add a title for the colums, something like "available" "selected" and maybe a simple explanation

Thank you very much for the efforts

**Comment by stephane-r** [comment]
Yes I agree that it is not explicit enough. The "move" state on the right of the selected playlists is not useful. I will adjust this part soon :)
